tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3997,4 +3997,118 @@ function data_set_spacing_sizes_when_invalid() {
 			),
 		);
 	}
+
+	/**
+	 * @ticket 56903
+	 */
+	function test_update_separator_declarations() {
```

Comment:
And also add the visibility identifier to all class methods

```suggestion
	public function test_update_separator_declarations() {
```